_config.yml

```diff
@@ -111,6 +111,7 @@ icon-tag:
   galaxy-home: fas fa-home
   galaxy-info: fas fa-info-circle
   galaxy-library: far fa-folder
+  galaxy-link: fas fa-link
```

Comment:
ok, I have added the icon.

So now you can use {% icon galaxy-link %} to insert the link icon in your FAQ.

(In case you are wondering how this works, the `fas fa-link` refers to the id of the icon in [fontawesome](https://fontawesome.com) )

Reply:
@Melkeb see above ^ . Let me know if you have any questions!

Reply:
@shiltemann Thank you. I will add the new icon